doc/user_guide/create_new_sale_channel.rst

```diff
@@ -0,0 +1,71 @@
+New Sale Channel
+=================
+
+This module helps you understand how to create a new sale channel.
+
+New Module 
+-----------
+
+On the top, click on the ``Create a new record`` button.
+
+.. figure:: images/23.png
+
+In the window that pops up, you can see a number of fields.
+These are the fields on New Record Window:
+
+* **Name**: The name of your new sales channel. This field is manadatory.
+* **Code**: Optional Field to easily lookup channels if you have many. 
+* **Company**: This is mandatory field to fill. Each channel has to be
+  associated with a company.
+* **Address**: If the channel represents a physical location, use the address
+  field to add an address. You can only choose from the addresses of your
+  company. This is optional field.
+* **Source**: How your channel will be handled. By default it is set to Manual
```

Comment:
The source of orders for the channel. The standard module only has a manual option which is used when your users key in orders from the Tryton sales modules. Downstream modules implement marketplaces like Amazon and eCommerce channels like Magento and Shopify